pagopa/pn-frontend
# PN-13445 - Create digital domicile activation page

## Short description
Implement a new page for digital domicile activation using the PnWizard component

## List of changes proposed in this pull request
- Added a new page with the route `/recapiti/attivazione-domicilio-digitale`

## How to test
Start PF/PG and then

1. Navigate to `/recapiti/attivazione-domicilio-digitale`

2. Verify that PnWizard is shown with all the instructions for activating the digital domicile (SERCQ SEND or PEC)

3. Click the "Attiva" button:
    - Ensure that the ToS are accepted.
    - Verify that SERCQ SEND is successfully enabled

Also, verify the scenario where the user already has a default PEC enabled. In this case, the wizard should exclude the PEC section and display a banner above the ToS copy
